src/sdsc/parser.py

```diff
@@ -17,22 +17,33 @@
 #
 
 from lxml import etree
+from .xslt import (counttokens,
+                   # buildtermdata,
```

Comment:
Because this is WIP. It is quite tricky to resolve all the global dependencies. As I've wrote in the comment above, maybe a class could solve this issue.